ledger/src/blockstore_meta.rs

```diff
@@ -526,6 +526,19 @@ impl OptimisticSlotMetaVersioned {
         }
     }
 }
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Serialize, Deserialize)]
+pub struct SizeDetails {
+    pub last_fec_set_index: u32,
+    pub num_coding_received: usize,
+    pub num_data_received: usize,
```

Comment:
Can you leave some comments what constitutes as "received"? with respect to different routes you might obtain a shred: turbine, repair, erasure recovery.